.env.example

```diff
@@ -2,3 +2,7 @@ DB_NAME="database name"
 DB_PASSWORD="database password"
 DB_USER="username"
 DB_HOST="hostmae"
+NODE_ENV = production
```

Comment:
@bright258 NODE_ENV should be development not production

Reply:
Okay, I have made the correction